test/unit/Reflection/Adapter/ReflectionTypeTest.php

```diff
@@ -52,7 +53,7 @@ public function methodExpectationProvider(): array
      */
```

Comment:
It's line 38 here failing, any thoughts on how to solve this @kukulich @Ocramius ?

Reply:
All adapters have to redeclare all methods from core classes: https://github.com/Roave/BetterReflection/pull/765#discussion_r711575207

There was bug fixed in: https://github.com/Roave/BetterReflection/pull/769

So your adapter does not redeclare some methods.

The assertion should probably have some message to be clear...

Reply:
Is this still relevant after rebase?